src/index.js

```diff
@@ -69,10 +69,6 @@ exports.encode = function encode (digest, code, length) {
     throw new Error('digest length should be equal to specified length.')
   }
 
-  if (length > 127) {
```

Comment:
It's currently not using a varint, but I'm unclear from the multihash spec what exactly to use as this length uses a fixed size of 4 bytes.